docs-gen/layouts/partials/hooks/head-end.html

```diff
@@ -0,0 +1 @@
+<base href="{{ .Site.BaseURL }}">
```

Comment:
Fixing [the invalid resource paths](https://github.com/sky-uk/kfp-operator/pull/373#issuecomment-2407590392) by making all paths relative (setting `canonifyURLs` to false), meant that images could no longer be found - they needed to be referenced under `/kfp-operator/images/...etc`. Adding the baseURL in the head tag like we're doing in this file was [one recommended way of fixing this](https://discourse.gohugo.io/t/best-way-to-reference-an-image-in-static-directory-when-baseurl-could-contain-a-sub-directory/15461/3), and allows us to still reference images in markdown like so:
```markdown
![cd-ct](images/cd-ct.svg)
```